h2o-algos/src/main/java/hex/tree/SharedTreeModelWithContributions.java

```diff
@@ -20,7 +22,9 @@
         P extends SharedTreeModel.SharedTreeParameters,
         O extends SharedTreeModel.SharedTreeOutput
         > extends SharedTreeModel<M, P, O> implements Model.Contributions {
-    
+
+  private transient final ContributionComposer contributionComposer = new ContributionComposer();
```

Comment:
do you need this to be an instance variable? I think this should be avoided because you can lose the instance on serialization and deserialization if that ever happens

just make a new instance when you need it

Reply:
Fixed

Reply:
I just noticed that the field is transient, its not harm to make instance if its needed

Reply:
@valenad1 was it fixed or was it not fixed? Is still see the transient field defined in this class.

Please read my original comment. You are defining a transient field on a class that can be serialized. In that case, you will lose the value of the instance variable if serialized and deserialized.

Reply:
Yes, it is fixed in 66fecf09d262fb1ab87ea714000854ab42400e6c